### PR TITLE
Add support for multiple fonts

### PR DESCRIPTION
**Pull request by RoelN**

### CODE REVIEW

File: codeGeneration.js [hunk 1]
```diff
@@ -125,24 +151,26 @@ const buildBodyRule = fontData => {
...
+  } else {
+    root.append(buildRegularStyles(fontData));
+  }
 
   return root;
 };
 
 const buildFontJs = fontData => {
-  return `export const fontName = "${fontData.name}";`;
+  return `fontNames.push("${fontData.name}");\n`;
```
Comment (pascalw) [inline codeGeneration.js:167]: How does this work? It seems like `fontNames` is not declared, nor is it exported?

**Review by pascalw — COMMENTED** [review]
Nice work @RoelN! 👍

**Comment by pascalw** [comment]
I see this project is using Travis CI still, back from when it was a private repo that could not use GitHub Actions. I'll change this to GitHub Actions.

File: codeGeneration.js (hunk 1 above)
Comment (RoelN) [reply codeGeneration.js:167]: It's not very clear from this code, but it ties into this: https://github.com/kabisa/specimen-skeleton/pull/58/files#diff-076df69e37321e03eb2edb6809afca0bR116

This way I don't have to keep track in this function if it's the first font, or the second/third/etc.

But I'm open to suggestions if this needs improvement!

**Review by pascalw — APPROVED** [review]

**Pull request merged**